src/types.ts

```diff
@@ -24,7 +24,7 @@ export interface ResponseProofGenKey extends ResponseBase {
 }
 
 export interface ResponseSign extends ResponseBase {
-  signatures?: Buffer[]
+  signatures?: Buffer
```

Comment:
Should it be called "signature" instead?

Reply:
Good catch 😉 Fixed!